### ISSUE TITLE
Comic bulldozer mode

### ISSUE THREAD

**Issue by kanryu** [kanryu/quickviewer#107]
QuickViewer's emphasis on image switching comfort is to ensure that you don't feel stressed when viewing a lot of images.
As a result, using QuickViewer, hundreds of digital scraps or comics can be opened one after another.
However, I think that the current GUI design is not suitable for the task of checking the contents while switching many folders and archives one after another.

- when maximizing the window, the shared area of the attached bar is too large
- The button to switch Volume is not displayed by default in full screen, and the name of the currently open Volume can not be known (when displaying full screen text)

In other words, a simple mode for full screen is required to display only the necessary GUI such as current Volume and image name and file path, next page, previous page, next volume, previous volume, etc. You
